docs/development/setup_development.rst

```diff
@@ -20,21 +20,36 @@ Quick Start
 The Docker based environment is suitable for developing the web application
 and updating the documentation.
 
+Ensure Git is installed on your system. Use package management tools to update your local package index. Once the update is complete, you can download and install Git:
 
 Ubuntu or Debian GNU/Linux
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
+Run the following commands to update the packages and to install from the default Git repositories:
 
 .. code:: sh
 
    sudo apt-get update
    sudo apt-get install -y make git
 
+
+Fedora and Other Linux Distributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. note:: To install Docker Engine, you need the 64-bit version of Fedora 30 or higher.
+
+Run the following commands to update the packages and to install from the default Git repositories:
```

Comment:
Same edit as above. As noted in another comment, just running `sudo dnf install git` will implicitly update the package index, while `sudo dnf update` updates all installed packages and is deprecated.